src/NodaTime/XmlSchemaDefinition.cs

```diff
@@ -0,0 +1,241 @@
+// Copyright 2020 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+using NodaTime.Extensions;
+
+namespace NodaTime
+{
+    /// <summary>
+    /// Provides XML schema types.
+    /// </summary>
+    public static class XmlSchemaDefinition
```

Comment:
Does this actually need to be public? I haven't looked at the demo code yet - does it need to refer to this?

Reply:
Yes, both `XmlSchemaDefinition.NodaTimeXmlNamespace` and `XmlSchemaDefinition.NodaTimeXmlSchema` need to be accessed in the demo code. These are the only two Xml schema related public properties. Everything else is internal or private.

Reply:
(Please don't resolve a conversation until we've reached a consensus - it makes it hard to find things that still need discussion.)

Okay, if it needs to be public, I'd really like it to be out of the NodaTime namespace, given that it's not going to be significant for most users. I don't mind adding a NodaTime.Xml namespace for this - we could even potentially move the static NodaTime.DateTimeZoneProviders.Serialization property into a type within that namespace as well, to be clearer.

Reply:
Sorry, I used _resolve conversation_ for me to keep track of what I replied to, I didn't realise that the conversation would be marked as resolved for everyone.

I moved these two properties in a new `NodaTime.Xml` namespace. I haven't moved the `NodaTime.DateTimeZoneProviders.Serialization` into that new namespace as part of this pull request though. We can probably do it separately. Also, this is a breaking change but maybe you don't mind since you are going to release a new major version?

Reply:
Definitely best to keep the "moving to a different namespace" for a separate PR - I'll take care of that after this one's in. And yes, it's a breaking change - I'll document it when I make the change.